lib/rtp_h264/nal_header.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Membrane.Element.RTP.H264.NALHeader do
```

Comment:
Naming is inconsistent - there's `NALHeader` and `FU.Header`

Reply:
The problem is:
 - There is no NAL module, why would I do NAL.Header
 - Why should I name FUHeader if there is FU module and FU.Header is used only in FU (so it is clearly submodule)

Can you suggest a problem to this dilema?